rust/Cargo.toml

```diff
@@ -18,6 +18,17 @@
 name = "rocketmq"
```

Comment:
We currently do not specify the version of the Rust toolchain in the workflow of the Github Action. In fact, the Rust version used in the default runner environment is relatively high(see https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#rust-tools), so compilation is always no problem. However, I did encounter compatibility issues when compiling under the 1.63 version of the Rust toolchain, and the problem was not solved until I switched to 1.68.2.

I think it would be best to specify the [rust-version](https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field) in `Cargo.toml` to determine the minimum version of the Rust toolchain required(e.g. https://github.com/assert-rs/predicates-rs/blob/660f658285702e0219d9d76e65881ea5b8536a2c/Cargo.toml#L8).

In addition, could you help to make it clear in the documentation and GitHub Action workflow what our version requirements are?

Reply:
I have set `edition = "2021"` in `Cargo.toml`. It limits the minimum version of rust to 1.56.0.

Reference: https://doc.rust-lang.org/edition-guide/rust-2021/index.html

Reply:
Nice! I think it is a great choice for a Rust lib to have a low requirement on the version of Rust toolchain.

Meanwhile, `edition` and `rust-version` seem to have different semantics. The former focuses on the major features of Rust, while the latter is directly facing the compiler. So perhaps they cannot be replaced with each other? We also need to make it clear to our developers what is the minimum version required for current client build. From my current attempts, version 1.56 does not guarantee the success of project build.

see: https://github.com/aaron-ai/rocketmq-clients/actions/runs/4740564022/jobs/8416534825

Reply:
I have set `rust-version` to `1.61` which is this crate's msrv and tested it on my Mac.

<img width="658" alt="image" src="https://user-images.githubusercontent.com/32436597/233018688-35886f2b-d794-456c-8bfb-92ecdeb956cc.png">

Reply:
Fantastic work, our client now has an MSRV! Looking forward to you configuring the MSRV in our Github Action workflow in the next pull request to avoid potential compatibility issues that may be inadvertently introduced during the community development process.